ws-server/src/server.rs

```diff
@@ -228,6 +239,7 @@ async fn background_task(
 			send_error(id, &tx, code.into());
 		}
 	}
+	Err(Error::Custom("Server is terminated".to_string()))
```

Comment:
Why is it an error? Logically, stopping the server is the normal way to exit the function.
And now, if I didn't miss anything, this function never returns an `Ok` value.

Reply:
good point, will change